Source/DafnyLanguageServer.Test/Various/ConcurrentInteractionsTest.cs

```diff
@@ -99,66 +98,66 @@ decreases y
       // The original document contains a syntactic error.
       var initialLoadDiagnostics = await diagnosticsReceiver.AwaitNextDiagnosticsAsync(CancellationTokenWithHighTimeout, documentItem);
       await AssertNoDiagnosticsAreComing(CancellationTokenWithHighTimeout);
-      Assert.AreEqual(1, initialLoadDiagnostics.Length);
+      Assert.Single(initialLoadDiagnostics);
 
       ApplyChange(ref documentItem, new Range((2, 1), (2, 1)), "\n}");
 
       // Wait for resolution diagnostics now, so they don't get cancelled.
       // After this we still have never completing verification diagnostics in the queue.
       var parseErrorFixedDiagnostics = await diagnosticsReceiver.AwaitNextDiagnosticsAsync(CancellationTokenWithHighTimeout, documentItem);
-      Assert.AreEqual(0, parseErrorFixedDiagnostics.Length);
+      Assert.Empty(parseErrorFixedDiagnostics);
 
       // Cancel the slow verification and start a fast verification
       ApplyChange(ref documentItem, new Range((0, 0), (3, 1)), "function GetConstant(): int ensures false { 1 }");
 
       var verificationDiagnostics = await diagnosticsReceiver.AwaitNextDiagnosticsAsync(CancellationTokenWithHighTimeout, documentItem);
-      Assert.AreEqual(1, verificationDiagnostics.Length);
+      Assert.Single(verificationDiagnostics);
 
       await AssertNoDiagnosticsAreComing(CancellationTokenWithHighTimeout);
     }
 
     /// <summary>
     /// If this test is flaky, increase the amount of lines in the source program
     /// </summary>
-    // [TestMethod, Timeout(MaxTestExecutionTimeMs)]
-    public async Task ChangeDocumentCancelsPreviousResolution() {
-      string CreateCorrectFunction(int index) => @$"function GetConstant{index}(x: int): int {{ x }}";
-
-      var functionWithResolutionError = "function GetConstant(): int { x }\n";
-      var slowToResolveSource = functionWithResolutionError + string.Join("\n", Enumerable.Range(0, 1000).Select(CreateCorrectFunction));
-      var documentItem = CreateTestDocument(slowToResolveSource, "veryLongDocument.dfy");
-      client.OpenDocument(documentItem);
-
-      // Change but keep a resolution error, cancel previous diagnostics
-      ApplyChange(ref documentItem, new Range((0, 30), (0, 31)), "y");
-
-      // Fix resolution error, cancel previous diagnostics
-      ApplyChange(ref documentItem, new Range((0, 30), (0, 31)), "1");
-
-      var resolutionDiagnostics = await diagnosticsReceiver.AwaitNextDiagnosticsAsync(CancellationToken, documentItem);
-      Assert.AreEqual(0, resolutionDiagnostics.Length);
-
-      var verificationDiagnostics = await diagnosticsReceiver.AwaitNextDiagnosticsAsync(CancellationToken, documentItem);
-      Assert.AreEqual(0, verificationDiagnostics.Length);
-
-      await AssertNoDiagnosticsAreComing(CancellationToken);
-    }
-
-    [TestMethod, Timeout(MaxTestExecutionTimeMs)]
+    // [Fact(Timeout = MaxTestExecutionTimeMs)]
```

Comment:
Why didn't this one work?

Reply:
It's turned off on master, and when I turned it on it failed, so keeping that like it is in this PR

Reply:
Oh I missed that just the attribute was already commented out.

You can do `[Fact(Timeout = MaxTestExecutionTimeMs, Skip = "Not working")]` instead. That way it still shows up in the test run and we don't completely forget about it.